about.html

```diff
@@ -0,0 +1,76 @@
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="body">
+        <div class="socials">
+            <li><a href=""><img src="images/facebook.png" alt="facebook"></a></li>
+            <li><a href=""><img src="images/twitter.png" alt="twitter"></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My page</a></li>
+            <li><a href="">Log Out</a></li>
+        </div>
+    <header>
+        <nav class="mobile-nav">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+            <a href=""><img src="images/x-menu.png" alt="logo" width="20px" height="20px"></a>
+        </nav>
+        <nav class="desktop-nav">
+            <a href="#"><img src="images/logo.png" alt="x-menu" width="50px" height="50px"></a>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+            <div class="mobile-menu-icon">
+              <a href="#">
+                <img src="images/menu.svg" alt="logo" width="30px" height="30px">
+              </a>
+            </div>
+        </nav>
+    </header>
+    <h2 class="welcome-about-us">
+        Welcome to Course-R Droid, where learning becomes fun!
+    </h2>
+    <div class="information">
+        Course-R Droid is a technical school where you get to learn any form of technology that interest you from the hardware to the software part of any technological system.
+    </div>
+    <p class="contact-p">Please contact us via email for further questions about enrollment procedures at oguzieibehrichard@gmail.com.</p>
+    <div class="about-us-logo">
+        <img src="images/logo.png" alt="logo">
+        <p>Course-R Droid was founded by a young student who is currently the CEO of this great institution Mr Richard Oguzie-Ibeh, He and his friends started this establishment from a very young age of 19.</p>
+    </div>
+    <h6>Meet our world best International graduates</h6>
+    <div class="graduates">
+        <div class="graduate-1">
+            <img src="images/graduate-1.jpg" alt="1">
+            <h5>Michael Sam 2022</h5>
+        </div>
+        <div class="graduate-2">
+            <img src="images/graduate-2.jpg" alt="4">
+            <h5>Okoro Joshua 2016</h5>
+        </div>
+    </div>
+</div>
+<footer class="partners">
+    <h4>Partners</h4>
+    <hr>
+    <ul>
+        <li><a href="#"><img src="images/adidas.png" alt="adidas"></a></li>
+        <li><a href="#"><img src="images/fedex.png" alt="fedex"></a></li>
+        <li><a href="#"><img src="images/oracle.png" alt="oracle"></a></li>
+    </ul>
+</footer>
+<script src="script.js"></script>
+</body>
+</html>
```

Comment:
Good job adding the images for your graduates but id advise you to resize the images so the look like a rectangle just like in the design and it would be nice if you added the orange shadow on the images just as it is in the design template please stick to the design and pay more attention to detail and again you dont have a footer there should be a footer after ypur partners section please add this.


This is your design:
![about 4](https://user-images.githubusercontent.com/79506105/196391527-4682259f-1f51-4ecd-9ecf-4b94610e34f5.png)

Your design should look similar to this: 
![About last page ](https://user-images.githubusercontent.com/79506105/196391629-07c18e32-c264-483e-8da0-1bf4fea2c8f5.png)

Kindly note all changes should also  be done in the mobile view.

Reply:
Hmmmmn..., I was actually thinking about that too, Thank you for pointing that out 🥇